boards/openhwgroup/cv32a6_genesys_2/cv32a6_genesys_2_defconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+CONFIG_BASE64=y
+CONFIG_INCLUDE_RESET_VECTOR=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_NS16550_ACCESS_WORD_ONLY=y
+CONFIG_CONSOLE_HANDLER=y
+CONFIG_XIP=n
+CONFIG_INIT_STACKS=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=25000000
+CONFIG_FPU=y
```

Comment:
why is fpu needed for hello world for example?

Reply:
Looks like I misunderstood what the config does, I removed it.